hail/python/hail/ir/table_ir.py

```diff
@@ -934,8 +934,9 @@ def __init__(self, child, config):
 
     def _handle_randomness(self, uid_field_name):
         if uid_field_name is not None:
-            raise FatalError('TableToTableApply does not support randomness in consumers')
-        child = self.child.handle_randomness(None)
+            if self.config['name'] != 'TableFilterPartitions':
```

Comment:
why doers table filter partitions support random consumers?

Reply:
why not? super easy to pass through row UIDs.